server/plugin.go

```diff
@@ -175,6 +175,17 @@ func (p *Plugin) Reconfigure() error {
 	}
 
 	if err = filesBackend.TestConnection(); err != nil {
+		pluginConfig := p.Client.Configuration.GetPluginConfig()
+
+		// Disable the S3 settings in case the AWS config is invalid
+		pluginConfig["amazons3bucketsettings"].(map[string]interface{})["Enable"] = false
```

Comment:
@raghavaggarwal2308  Since the `fileBackend` remains the invalid S3 config, the plugin will still fail as it proceeds.  I think we need to re-fetch the settings and create another non-S3 backend here.

Reply:
@wiggin77 Since the AWS config is disabled. The filesbackend will update its value with the MM file storage config on line `163`. I have also tested this and its working fine. Please let me know your thoughts.